### PR TITLE
feat(community): Add property extraction for Nodes and Relationships in LLMGraphTransformer

### PR DESCRIPTION
**Pull request by gvasilei**
Adds support for extracting Node and Relationship properties on `LLMGraphTransformer`, similar to the Python version. A key distinction is that arbitrary property extraction is not supported. The caller needs to specify the property keys in advance.  

This is my first contribution here, so please let me know if I've missed anything, or if there is any feedback. Thank you!

### CODE REVIEW

**Comment by jacoblee93** [comment]
Thank you! Tagging in @easwee or @tomasonjo to have a look

File: libs/langchain-community/tsconfig.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "rootDir": "./src",
     "target": "ES2021",
     "lib": ["ES2021", "ES2022.Object", "DOM"],
-    "module": "ES2020",
-    "moduleResolution": "nodenext",
+    "module": "NodeNext",
```
Comment (tomasonjo) [inline libs/langchain-community/tsconfig.json:8]: Probably don't want to change tsconfig file

**Review by tomasonjo — APPROVED** [review]
LGTM

File: libs/langchain-community/tsconfig.json (hunk 1 above)
Comment (gvasilei) [reply libs/langchain-community/tsconfig.json]: Indeed! I was using Typescript 5.2 locally and faced an issue with module being `ES2020`. Reverted.

Comment (jacoblee93) [reply libs/langchain-community/tsconfig.json]: Yeah, you have to use TS 5.1. It's bad and we need to fix it at some point.

Comment (gvasilei) [reply libs/langchain-community/tsconfig.json]: Yes, that would be good. In any case, I've reverted that particular change. @jacoblee93 do you see any other areas for feedback?

**Comment by jacoblee93** [comment]
Thank you both!

**Pull request merged**